pyproject.toml

```diff
@@ -42,3 +42,9 @@ exclude_dirs = ["/venv/"]
 
 [tool.bandit.assert_used]
 skips = ["*/*test.py", "*/test_*.py"]
+
+[tool.mypy]
+ignore_missing_imports = true
```

Comment:
I think this doesn't match what's defined in https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#type-hints

Reply:
I sync'ed it with our guide but I still need to keep ignore_missing_imports for the ingress library